### PR TITLE
Jy/hero feature

### PR DESCRIPTION
**Pull request by theroadeldorado**
## Trello Card(s)
https://trello.com/c/nITszezT/18-section-name-this-section

## What was fixed?
Adds Hero Feature Paragraph and lots of site setup

## Screenshots of fix (visual changes)
Screen Shot 2019-11-19 at 1 21 00 PM
Screen Shot 2019-11-19 at 1 21 07 PM

### CODE REVIEW

File: web/themes/fire/assets/styles/base/_variables.scss [hunk 1]
```diff
@@ -5,6 +5,9 @@
 // z-indexes
 $z-index-admin-header: 10001;
 
+$font-family: 'Fira Sans', sans-serif;
```
Comment (1forh) [inline web/themes/fire/assets/styles/base/_variables.scss]: Bootstrap gives you a couple of variables you can use for this.

File: web/themes/fire/assets/styles/components/_buttons.scss [hunk 2]
```diff
@@ -13,4 +13,21 @@ a {
...
 
 .btn {
+  text-transform: uppercase;
+  font-weight: bold;
+
+  &--primary {
+    @include button-variant($secondary, transparent, $accent);
+  }
+
+  &--outline {
```
Comment (1forh) [inline web/themes/fire/assets/styles/components/_buttons.scss]: Are there any more outline variations? If so, maybe this should be `outline-white`

File: web/themes/fire/templates/components/hero-feature/_hero-feature.scss [hunk 3]
```diff
@@ -0,0 +1,116 @@
+.hero-feature {
+  position: relative;
+
+  &__background-image {
+    background-position: center center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
```
Comment (1forh) [inline web/themes/fire/templates/components/hero-feature/_hero-feature.scss]: You don't need to add the browser prefixes. I believe these are handled with our webpack.config but regardless, i believe all browsers support these now a days.

File: web/themes/fire/templates/components/hero-feature/_hero-feature.scss [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    @include media-breakpoint-down(md) {
+      @include make-col(12);
+      @include make-col-offset(0);
+      min-height: 0px;
+    }
+  }
+
+  &__image {
+    width: auto;
+    height: 400px;
```
Comment (1forh) [inline web/themes/fire/templates/components/hero-feature/_hero-feature.scss]: we should probably use the `maintain-ratio` mixin for this

File: web/themes/fire/templates/components/hero-feature/_hero-feature.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    }
+  }
+
+  .non-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  // &__video {
```
Comment (1forh) [inline web/themes/fire/templates/components/hero-feature/_hero-feature.scss]: need this?

File: web/themes/fire/templates/includes/video-modal/video-modal.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import $ from 'jquery';
+
+import { FireComponent, FireHelpers } from '@base';
+
+export class VideoModal extends FireComponent {
+  constructor(id) {
+    // register the component by calling `super()` and passing in the component id
```
Comment (1forh) [inline web/themes/fire/templates/includes/video-modal/video-modal.js]: can remove these comments when making these. Do you have the snippet to make these?

File: web/themes/fire/templates/includes/video-modal/video-modal.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+  get elements() {
+    return {
+      $modalID: this.$component.attr('id'),
+      $mobileURL: this.$component.find('[data-mobile]').attr('data-mobile'),
+    };
+  }
+
+  _initModal() {
+    let target = $(`#${this.elements.$modalID}`);
```
Comment (1forh) [inline web/themes/fire/templates/includes/video-modal/video-modal.js]: If `target` should be immutable, you can use `const` for this. Same with other variables.

File: web/themes/fire/templates/components/hero-feature/_hero-feature.scss (hunk 4 above)
Comment (theroadeldorado) [reply web/themes/fire/templates/components/hero-feature/_hero-feature.scss]: This image is an svg and can have any width.

**Pull request merged**